draft-irtf-qirg-principles-09-rc.xml

```diff
@@ -337,6 +359,14 @@
 
        <t>NOT (a |0> + b |1>) -> a |1> + b |0>.</t>
 
+       <t>It is important to note that "qubit" can have two meanings. In the
+       first meaning, "qubit" refers to a quantum *system* whose quantum state
+       can be expressed as a superposition of two basis states, which we often
+       label |0> and |1>. In the second meaning, "qubit" refers to the quantum
+       *state* of a quantum system with such two basis states. Both meanings
+       are used interchangeably in literature and the meaning is generally
+       clear from the context.</t>
```

Comment:
I can't quite follow what the distinction is.  The two meanings sound extremely similar to me.  Do you mean one meaning refers to the mathematical abstraction whereas the other meaning refers to the physical implementation?